src/utils/convert.js

```diff
@@ -83,20 +83,32 @@ ${props.join(',\n')}
 };
 
 const renderSampleField = (def) => {
-  const type = typesMap[def.type];
+  const type = typesMap[def.type] || 'string';
 
-  return `      ${def.key}: {
+  if (def.label) {
```

Comment:
Perhaps:

```js
return JSON.dump(_.mapValues(_.merge({}, def, {type}), (value) => quote(value)), null, 2);
```

Note that should they have extra keys than we needed they'll be included. I wonder if we should permit it.